### PR TITLE
*: Support replace stmt in new plan

### PR DESCRIPTION
**Pull request by shenli**
@qiuyesuifeng @coocood

### CODE REVIEW

File: executor/executor_write.go [hunk 1]
```diff
@@ -731,3 +736,93 @@ func getOnDuplicateUpdateColumns(assignList []*ast.Assignment, t table.Table) (m
 	}
 	return m, nil
 }
+
+// ReplaceExec represents a replace executor.
+type ReplaceExec struct {
+	InsertValues
+	fields   []*ast.ResultField
```
Comment (coocood) [inline executor/executor_write.go]: `fields` seems not used.

Comment (shenli) [inline executor/executor_write.go]: yes

**Comment by shenli** [comment]
@coocood  PTAL

**Comment by qiuyesuifeng** [comment]
LGTM

**Comment by coocood** [comment]
LGTM

**Pull request merged**